cves/CVE-2015-1831.yml

```diff
@@ -121,10 +124,10 @@ discovered:
     The "apache" flag can be true, false, or nil.
     If there is no evidence as to how this vulnerability was found, then you may
     leave the entries blank except for "answer". Write down where you looked in "answer".
-  answer:
+  answer: It was discovered by Jasper Rosenberg at Cargurus
   date:
   automated:
-  apache:
+  apache: 2.3.20
   contest:
 
 subsystem:
```

Comment:
The subsystem needs to be filled in for the Travis CI build to complete properly.  Look at the code to figure out what subsystem this fix relates to, helped by what directories the changed files reside in.

Reply:
Subsystem has been filled out